### ISSUE TITLE
Accessibility - custom aria labels or prop to support the addition of the day of the week during readout (ex. Tuesday September 30th 2019)

### ISSUE THREAD

**Issue by KevRyan2** [wojtekmaj/react-calendar#256]
Users utilizing a screenreader should be able to hear the day of the week called out along with the rest of the date that's read, but there isn't support for custom aria-labels or any prop that would allow this functionality.  This is an accessibility gap and would seem to be a small, non-breaking change that could be made. Would you be able to provide this functionality in an upcoming version?

**Comment by wojtekmaj** [comment]
Good catch! React-Calendar provides multiple date formatters by default and allow them to be overwritten, but not this particular one.

3f29262 adds `formatLongDate` overwrite, which will be used in every place that needs long date to be formatted (currently Day tile's abbr element).

Change will be released with the next React-Calendar 3.0 beta or final release, whichever comes first.
